adapters/beachfront/beachfront.go

```diff
@@ -16,12 +16,18 @@ import (
 const Seat = "beachfront"
 const BidCapacity = 5
 
-const BannerEndpoint = "https://display.bfmio.com/prebid_display"
+const BannerEndpoint = "http://display.bfmio.com/prebid_display"
+
+// const BannerEndpoint = "http://qa.bfmio.com/prebid_display"
```

Comment:
Any reason these are left commented, rather than just deleted?

Reply:
No real reason. I'll leave similar out in future PRs.